modules/navigation/register_types.cpp

```diff
@@ -53,6 +61,12 @@ NavigationServer3D *new_server() {
 
 void initialize_navigation_module(ModuleInitializationLevel p_level) {
 	if (p_level == MODULE_INITIALIZATION_LEVEL_SERVERS) {
+#ifdef MODULE_NAVIGATION_ENABLED
```

Comment:
Are these guards actually useful?  I thought the entire module wouldn't be compiled if the module was disabled.

Reply:
The guards are only useful to shutup the "Minimal Template" pr check that tries a build without navigation which is not possible at the moment.